### ISSUE TITLE
multiple news items have empty hyperlinks

### ISSUE THREAD

**Issue by StephDriver** [BirkbeckCTP/janeway#4083]
**Describe the bug**
Links are appearing twice, the second time with no link text so that visually they seem to appear once, but to assistive technologies there is a second link without discernible text.  The second link has whitespace rather than text.

It is unclear whether this is a human input issue, or a software one.  It may be that the person writing the news item put in a link, then though they'd removed it because all the text had been removed not realising the remaining whitespace was enough for the link to stay?   

Examples
### news pages - e.g. https://www.openlibhums.org/news/579/
- missing alt-text
- link does not have discernible text 
```
<p>
<i><a href="https://www.glossa-journal.org/" target="_blank">Glossa</a></i>
<a href="https://www.glossa-journal.org/" target="_blank"> </a>
publishes about 130 articles/year, so the work 
is rather demanding on a team of volunteer editors. 
A flawless and flexible submission system is a necessity. 
Janeway delivers.<br></p> 
```
link appears twice, once without text.

Likewise on https://www.openlibhums.org/news/425/

```
<p>If you like the work that the Open Library of Humanities is doing, 
please consider asking your institution to support us financially. 
We cannot operate without our library members. 
More details for libraries can be found here: 
<a href="https://www.openlibhums.org/plugins/supporters/signup" target="_blank">https://www.openlibhums.org/plugins/supporters/signup</a>
<a href="https://www.openlibhums.org/plugins/supporters/signup" target="_blank" style="background-color: rgb(255, 255, 255);"></a></p>
```

**Additional context**
This came from the testing of the new site with Axe, see BirkbeckCTP/hourglass#349

**Comment by StephDriver** [comment]
We think this is a problem with user content that was related to the old text editor. Looking at content created with the new text editor there is no repeat. 

Therefore, this is fixed for future news items. 

Finding all places where this was introduced by the old editor is beyond capacity for manual testing and requires automated testing of the live site. Automated testing is being considered for Phase 2 of the A11y project, and when it is implemented any remaining issues of this kind will be found.  

Therefore closing this for now as no further action required.
